lib/types/student.rb

```diff
@@ -10,18 +10,19 @@ class Student < Base
                   :provider,
                   :email,
                   :grades,
-                  :tenant_id
+                  :school_id
 
       def initialize(attributes = {}, client: nil)
         @uid          = attributes['sourcedId']
         @first_name   = attributes['givenName']
         @last_name    = attributes['familyName']
         @status       = attributes['status']
         @email        = attributes['email']
+        @api_username = attributes['username']
```

Comment:
this looks like a regression on main that caused two specs to fail.   Restoring this attribute fixes it them.    There are still two other specs that fail on main that I could not quickly figure - they need the call to fetch schools to be mocked but it cannot be mocked the way other endpoints are mocked.. so giving up for now.